src/cli/commands/gsn.ts

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env node
 
 import commander from 'commander'
+import { version } from '../../../package.json'
```

Comment:
I remember that this was an issue before...

Reply:
It works with `npx gsn -V` from another project, as well as with `tsnode ./src/cli/commands/gsn.ts -V`  and with `node ./dist/cli/commands/gsn.js`
And of course, it works with the docker wrapper by running `dockers/gsn/gsn -V`

So I think its OK...